php/commands/src/CLI_Command.php

```diff
@@ -103,6 +103,10 @@ public function info( $_, $assoc_args ) {
 
 		$runner = WP_CLI::get_runner();
 
+		$system_os           = php_uname( 's' );
```

Comment:
Could you move this block before `$runner = WP_CLI::get_runner();` please?

Also might as well give as much info as possible so:
```
$system_os = sprintf( '%s %s %s %s', php_uname( 's' ), php_uname( 'r' ), php_uname( 'v' ), php_uname( 'm' ) );
```
(only leaving out host name `'n'` which people mightn't want exposed)?

Reply:
ok @gitlost  on it